src/SFA.DAS.ApprenticeCommitments.Web/Pages/SignOut.cshtml.cs

```diff
@@ -2,14 +2,15 @@
 using Microsoft.AspNetCore.Authentication.OpenIdConnect;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.Linq;
 
 namespace SFA.DAS.ApprenticeCommitments.Web.Pages
 {
     public class SignOutModel : PageModel
     {
         public IActionResult OnGet()
         {
-            foreach (var cookie in Request.Cookies.Keys)
+            foreach (var cookie in Request.Cookies.Keys.Where(x => x.Contains("Apprenticeships")))
```

Comment:
Is this purely to delete the "sign-in" cookie?  Might it be worth being explicit and only deleting ".Apprenticeships.Application"?

Reply:
Hi yes it was to delete the .Apprenticeships.Application cookie only - as currently it was deleting the optional cookies too.  Changed to .Apprenticeships.Application.